src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -1124,6 +1124,26 @@ const uint8_t ConnectionHandlerImpl::GetSessionIdFromSecondaryTransport(
   return 0;
 }
 
+Connection* ConnectionHandlerImpl::GetPrimaryConnection(
+    const ConnectionHandle connection_handle) const {
+  LOG4CXX_DEBUG(logger_,
+                "Getting primary connection for ID " << connection_handle);
+  ConnectionList::const_iterator it = connection_list_.find(connection_handle);
+  if (connection_list_.end() == it) {
```

Comment:
@AKalinich-Luxoft can we use `helpers::in_range` to make more clear & short code?

Reply:
@AByzhynar unfortunately, we need to have iterator here to get the pointer to appropriate connection in the code below, so we have to use `find` in this scope